docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -6,15 +6,62 @@ checked: true
 ---
 # Command Line Interface
 
+This topic provides information on how to run TestCafe tests from a command line.
+
+## Installation
+
+You can install TestCafe globally, or locally to your project.
+
+### Globally
+
+```bash
+npm install -g testcafe
+```
+
+After that, you can run TestCafe by using the `testcafe` command (see [Usage](#usage)).
+
+```bash
+testcafe -v
+```
+
+### Locally
+
+The following command will install TesCafe into your project directory and save it in the dependencies list.
+
+```bash
+npm install --save-dev testcafe
+```
+
+The local installation has two advantages:
+
+* It makes your project setup easier: running `npm install` in the project directory will automatically install TestCafe as well.
+* Different projects can depend on different versions of TestCafe.
+
+You can run the local version of TestCafe in one of the following ways.
+
+* From the `node_modules` folder.
+
+    ```bash
+    node node_modules/testcafe/bin/testcafe -v
```

Comment:
All locally installed binaries are linked to `node_modules/.bin`. So the preferred way to work with local binary is executing it `./node_modules/.bin/testcafe`. But, maybe we don't really need to point out how to use local binary in CLI: https://github.com/DevExpress/testcafe/issues/640#issuecomment-230416634. \cc @inikulin